.github/workflows/integration_test_charm.yaml

```diff
@@ -362,6 +368,11 @@ jobs:
           sg '${{ steps.parse-cloud.outputs.group }}' -c "microk8s.kubectl rollout status --namespace kube-system --watch --timeout=5m deployments/coredns"
           sg '${{ steps.parse-cloud.outputs.group }}' -c "retry --times 3 --delay 5 -- sudo microk8s enable hostpath-storage"
           sg '${{ steps.parse-cloud.outputs.group }}' -c "microk8s.kubectl rollout status --namespace kube-system --watch --timeout=5m deployments/hostpath-provisioner"
+          if [[ '${{ inputs.metallb-addon }}' = 'true' ]]
+          then
+            IPADDR=$(ip -4 -j route get 2.2.2.2 | jq -r '.[] | .prefsrc')
```

Comment:
Instead of installing jq and depends on it, I would do here :
`ip -4 route get 2.2.2.2 | awk '{print $7;exit}'`
or just sed if you do not want to depends on the order number:
`ip -4 route get 2.2.2.2 | sed -n -e 's/^.*src \([^ ]*\).*/\1/p'`

Reply:
use sed instead of jq in [b9acbe7](https://github.com/canonical/data-platform-workflows/pull/244/commits/b9acbe7d8b97ea37d322a1dbb0424f5ee303f2d9)